services/ui-src/src/components/rates/types/NDREnhanced.tsx

```diff
@@ -71,6 +71,17 @@ export const NDREnhanced = (props: PageElementProps<NdrEnhancedTemplate>) => {
       newErrorObject.rates[assessIndex!][fieldType] = errorMessage;
     }
 
+    if (parseNumber(newDisplayValue.denominator) === 0) {
+      for (const [index, rate] of newDisplayValue.rates.entries()) {
+        if (parseNumber(rate.numerator) !== 0) {
+          newErrorObject.rates[index].numerator = ErrorMessages.denomenatorZero(
+            "Numerator",
```

Comment:
Just out of curiosity, is there a reason Numerator is capitalized while denominator isn't?

Reply:
Its the actual hint text that the user sees.